{
  "CVE Code": "CVE-2022-23097",
  "Published": "2022-01-28T00:00:00",
  "Updated": "2024-11-19T19:10:45.191Z",
  "Description": "An issue was discovered in the DNS proxy in Connman through 1.40. forward_dns_reply mishandles a strnlen call, leading to an out-of-bounds read.",
  "ProblemTypes": [
    "n/a"
  ],
  "Affected": [
    {
      "vendor": "n/a",
      "product": "n/a",
      "versions": [
        "n/a"
      ]
    }
  ],
  "Reference": [
    "https://git.kernel.org/pub/scm/network/connman/connman.git/log/",
    "https://www.openwall.com/lists/oss-security/2022/01/25/1",
    "https://lists.debian.org/debian-lts-announce/2022/02/msg00009.html",
    "https://www.debian.org/security/2022/dsa-5231",
    "https://security.gentoo.org/glsa/202310-21"
  ]
}